tests/api_requests/test_smalltalk.py

```diff
@@ -0,0 +1,29 @@
+import samanthas_telegram_bot.api_queries.smalltalk as smalltalk
+from samanthas_telegram_bot.data_structures.constants import ALL_LEVELS
+from samanthas_telegram_bot.data_structures.enums import SmalltalkTestStatus
+from tests.constants import TEST_DIR
+
+API_TEST_DIR = TEST_DIR / "api_requests"
+
+
+DATA_FILES = tuple((API_TEST_DIR / "smalltalk_json").glob("*.json"))
+
+
+def test_process_smalltalk_json():
+    for file in DATA_FILES:
```

Comment:
I suggest to use pytest.mark.parametrize here

Reply:
I was going to use it, but I thought I would use these rules based on file names. Some files undergo one check, some several checks. Not sure it's doable with `parametrize` and not sure it's worth changing just for the sake of using it. I'm a huge fan of `parametrize` otherwise, don't get me wrong :)

Reply:
Maybe then split it to multiple tests, one test for each file or something like that?

Reply:
So far it's hard to understand what the motivation is for changing the test - it seems convenient calling test files according to a certain convention and have the tests pick them up automatically

Reply:
Split in two tests, each one with `pytest.mark.parametrize`